Dockerfile

```
@@ -110,7 +112,9 @@ RUN wget https://ffmpeg.org/releases/ffmpeg-7.1.tar.xz \
     && make -j$(nproc) \
     && make install \
     && cd .. \
-    && rm -rf ffmpeg-7.1 ffmpeg-7.1.tar.xz
+    && rm -rf ffmpeg-7.1 ffmpeg-7.1.tar.xz \
+    && cp -P /usr/lib/*/libvpx.so* /opt/ffmpeg/lib/ 2>/dev/null || \
+       cp -P /usr/lib/libvpx.so* /opt/ffmpeg/lib/ 2>/dev/null || true
```

error out the build if not found?
```suggestion
    && rm -rf ffmpeg-7.1 ffmpeg-7.1.tar.xz \
    && cp -P /usr/lib/*/libvpx.so* /opt/ffmpeg/lib/ 2>/dev/null || \
       cp -P /usr/lib/libvpx.so* /opt/ffmpeg/lib/ 2>/dev/null || { echo "Error: libvpx.so not found"; exit 1; }
```